pkg/search/sortfields/fields_map.go

```diff
@@ -19,6 +19,14 @@ var (
 				},
 			}
 		},
+		search.PolicyCategoryName: func(option *v1.QuerySortOption) []*v1.QuerySortOption {
```

Comment:
This is a noop `PolicyCategoryName` to `PolicyCategoryName`

Reply:
https://github.com/stackrox/stackrox/pull/2289